Doppler.Currency/Startup.cs

```diff
@@ -79,21 +75,6 @@ public void ConfigureServices(IServiceCollection services)
                 });
         }
 
-        private void AddServiceSettings(IServiceCollection services)
-        {
-            var dofSettings = new CurrencySettings();
-            Configuration.GetSection("CurrencyCode:DofService").Bind(dofSettings);
-            services.AddSingleton(dofSettings);
-
-            var bnaSettings = new CurrencySettings();
-            Configuration.GetSection("CurrencyCode:BnaService").Bind(bnaSettings);
-            services.AddSingleton(bnaSettings);
```

Comment:
How are configured this services now?
I do not fine any other reference in this PR.